### PR TITLE
chore(icons): Update and add missing pastanaga icons from https://pastanaga.io/icons ...

### PR DESCRIPTION
**Pull request by nileshgulia1**
...along with docs pertaining to them.

### CODE REVIEW

**Comment by nileshgulia1** [comment]
Fixes #3207

File: docs/source/developer-guidelines/icons.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+html_meta:
+  "description": "Icons used in Volto"
+  "property=og:description": "Icons used in Volto"
+  "property=og:title": "Icons"
+  "keywords": "Volto, Plone, frontend, React, Icons, svg, pastanaga"
+---
+
+# Icons
+
+Volto has pre-defined set of SVG icons from Pastanaga UI icon system and are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/icons.md:11]:
```suggestion
Volto has a pre-defined set of SVG icons from the Pastanaga UI icon system.
They are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).

The following example shows how to display one of these icons.```

File: docs/source/developer-guidelines/icons.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+```js
+import addUserSVG from '@plone/volto/icons/add-user.svg';
+import { Icon } from '@plone/volto/components';
+
+<Icon name={addUserSVG} size="24px" />;
+```
+
+```{note}
+These icons are intended to be used only in "official" Pastanaga UI, so please refrain to use them on personal projects unless they are based on Pastanaga UI.
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/icons.md:21]:
```suggestion
These icons are intended to be used only in the "official" Pastanaga UI.
Please refrain from using them on personal projects unless they are based on the Pastanaga UI.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor English grammar fixes, addition of a sentence to introduce a code example.

File: docs/source/developer-guidelines/icons.md (hunk 2 above)
Comment (nileshgulia1) [reply docs/source/developer-guidelines/icons.md]: @stevepiercy Done!

**Review by tiberiuichim — APPROVED** [review]

File: src/icons/list-arrows.svg [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36">
-  <g fill-rule="evenodd">
-    <polygon points="11 11 33 11 33 8.999 11 8.999"/>
-    <polygon points="11 19 33 19 33 17 11 17"/>
-    <polygon points="11 27 33 27 33 25 11 25"/>
-    <polygon points="5.091 7 8 10 5.091 13 4 11.875 5.818 10 4 8.125"/>
-    <polygon points="5.091 15 8 18 5.091 21 4 19.875 5.818 18 4 16.125"/>
-    <polygon points="5.091 23 8 26 5.091 29 4 27.875 5.818 26 4 24.125"/>
+  <g fill="none" fill-rule="evenodd">
+    <polygon fill="#000" points="11 11 33 11 33 8.999 11 8.999"/>
```
Comment (avoinea) [inline src/icons/list-arrows.svg:3]: @nileshgulia1 I wonder why is it filled with `#000`

Comment (avoinea) [reply src/icons/list-arrows.svg:3]: Ok, it seems broken in master https://github.com/plone/volto/blob/master/src/icons/verified.svg vs the one from your pr https://github.com/plone/volto/blob/edbee4f2366aa94b211ebbcbeadbba6ad64a1436/src/icons/verified.svg

**Review by avoinea — APPROVED** [review]

File: docs/source/developer-guidelines/icons.md [hunk 4]
```diff
@@ -8,7 +8,10 @@ html_meta:
 
 # Icons
 
-Volto has pre-defined set of SVG icons from Pastanaga UI icon system and are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).
+Volto has a pre-defined set of SVG icons from the Pastanaga UI icon system.
+They are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).
+
+The following example shows how to display one of these icons.```
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/icons.md:14]: It looks like some trailing backticks snuck in here.

```suggestion
The following example shows how to display one of these icons.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Looks like a typo.

File: docs/source/developer-guidelines/icons.md (hunk 4 above)
Comment (nileshgulia1) [reply docs/source/developer-guidelines/icons.md]: Done

**Review by sneridagh — APPROVED** [review]

**Comment by nileshgulia1** [comment]
@sneridagh Perhaps the inline "styled-fills" cannot be overridden. Thanks! Should be fixed in [e62f47b](https://github.com/plone/volto/pull/3208/commits/e62f47bdb77224f3c58a3b188ed8c48d0999d845)

**Comment by nileshgulia1** [comment]
But for svg's like `grid.svg` and `asterisk.svg`, we essentially need some styling as they are now broken without the fills:
https://github.com/plone/volto/blob/40f167eaefff18f18d7c1643eb5470cf92363432/src/icons/asterisk.svg

Also the one from master is broken: https://github.com/plone/volto/blob/master/src/icons/verified.svg

**Review by sneridagh — APPROVED** [review]

**Pull request merged**